app/assets/javascripts/application.js

```diff
@@ -1,12 +1,6 @@
-/*
- * = require vendor/jquery-1.11.3.min.js
- * = require d3
- * = require jquery-accessibleMegaMenu.js
- * = require utility.js
- * = require components.js
- * = require cookies.js
- * = require drawer.js
- * = require navDropdown.js
- */
-
-
+//= require jquery-accessibleMegaMenu.js
+//= require d3.v3.min
```

Comment:
oh yes sorry staging is broken for other reasons (I was experimenting with some "conditional uglifying") this works as-is - will push to staging to show you!